android/widgets/src/main/java/org/nativescript/widgets/BorderDrawable.java

```diff
@@ -804,6 +806,23 @@ public Drawable getDrawable() {
         return drawable;
     }
 
+    @Override
+    public void getOutline(@NonNull Outline outline) {
+        if (android.os.Build.VERSION.SDK_INT >= 21) {
+            Path backgroundPath = new Path();
+            float[] backgroundRadii = {
+                Math.max(0, borderTopLeftRadius), Math.max(0, borderTopLeftRadius),
+                Math.max(0, borderTopRightRadius), Math.max(0, borderTopRightRadius),
+                Math.max(0, borderBottomRightRadius), Math.max(0, borderBottomRightRadius),
+                Math.max(0, borderBottomLeftRadius), Math.max(0, borderBottomLeftRadius)
+            };
+            backgroundPath.addRoundRect(new RectF(getBounds()), backgroundRadii, Path.Direction.CW);
+            outline.setConvexPath(backgroundPath);
+        } else {
+            super.getOutline(outline);
```

Comment:
As far as I can see `getOutline` was added in API 21 so technically there is no `super` to call on lower API levels.
Better throw exception like `throw new IllegalStateException("Method supported on API 21 or higher");`

Reply:
Sorry for the delay, should be good now :)